support/Pipelines/Bbh/Ringdown.py

```diff
@@ -163,37 +163,43 @@ def start_ringdown(
         polynomial_order=polynomial_order,
     )
 
-    # Compute ringdown shape coefficients and function of time info
-    # for ringdown
-    with spectre_h5.H5File(str(fot_vol_h5_path), "r") as h5file:
-        if fot_vol_subfile.split(".")[-1] == "vol":
-            fot_vol_subfile = fot_vol_subfile.split(".")[0]
-        volfile = h5file.get_vol("/" + fot_vol_subfile)
-        obs_ids = volfile.list_observation_ids()
-        fot_times = np.array(list(map(volfile.get_observation_value, obs_ids)))
-        which_obs_id = np.argmin(np.abs(fot_times - match_time))
-
-        logger.info("Desired match time: " + str(match_time))
-        logger.info("Selected ObservationID: " + str(which_obs_id))
-        logger.info("Selected match time: " + str(fot_times[which_obs_id]))
-
-    match_time = fot_times[which_obs_id]
-
-    (
-        expansion_func_with_2_derivs,
-        expansion_func_outer_boundary_with_2_derivs,
-        rotation_func_with_2_derivs,
-    ) = functions_of_time_from_volume(
-        str(fot_vol_h5_path), fot_vol_subfile, match_time, which_obs_id
+    evaluated_fot_dict = functions_of_time_from_volume(
+        str(fot_vol_h5_path), fot_vol_subfile, match_time
     )
+    match_time = evaluated_fot_dict["MatchTime"]
+
+    # This section checks for functions of time in the dictionary and alters
+    # some values in the evaluated functions of time dictionary. The ringdown
+    # only requires the outer boundary expansion map so here we set the inner
+    # expansion map to the identity. We also set the translation map to 0 since
+    # translation is not supported in the transition to ringdown yet.
+    if (
+        evaluated_fot_dict.get("Expansion") == None
+        or evaluated_fot_dict.get("ExpansionOuterBoundary") == None
```

Comment:
You just overwrite the expansion below and never use the value in the fot_dict, so you probably only need to check the outer boundary